### PR TITLE
html-relative-links - Lindsay Jin

### PR DESCRIPTION
**Pull request by lindsay-jin**
Screen Shot 2024-02-05 at 3 16 52 PM

### CODE REVIEW

File: html-relative-links/contact-us/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li><a href="../index.html">Home Page</a></li>
+      <li><a href="../about-me/about-me.html">About Me</a></li>
+      <li><a href="//projects/projects.html">Projects</a></li>
```
Comment (shawnkost) [inline html-relative-links/contact-us/contact-us.html]: This should be `../` rather than `//`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small thing here Lindsay!

File: html-relative-links/contact-us/contact-us.html (hunk 1 above)
Comment (lindsay-jin) [reply html-relative-links/contact-us/contact-us.html]: thanks for catching that!

**Review by RobertGardner — COMMENTED** [review]
You merged this before it was Approved. Unfortunately, that messes up our LMS. Would you be able to make a trivial change, commit it, the create a new PR for me to approve? That will clean up LMS. Thanks!

**Pull request merged**